src/core/image-core.js

```diff
@@ -57,18 +57,25 @@ function _rowToImage(row) {
 
   const imagePath = row['image_path'];
 
+  imageObj.url = prefixImageWithGCS(imagePath);
+
+  return imageObj;
+}
+
+function prefixImageWithGCS(imagePath) {
+  if (imagePath === "") {
+    return "";
```

Comment:
Use either single or double quotes consistently throughout the project

Reply:
@cc-akarvinen